lib/SearchAndSort/SearchAndSort.js

```diff
@@ -645,7 +645,7 @@ class SearchAndSort extends React.Component {
     } = this.props;
     this.log('action', 'cleared search query');
     this.setState({ locallyChangedSearchTerm: '' });
-    this.transitionToParams({ query: '', ...extraParamsToReset });
+    this.transitionToParams({ query: '', qindex: '', ...extraParamsToReset });
```

Comment:
Hey @Dmytro-Melnyshyn 
Are you sure that this change is needed across all Folio apps that leverage `SearchAndSort`?

Reply:
Hey @Terala-Priyanka, this is a bug, it should be fixed everywhere.